utkdigitalinitiatives/TRACE
# Address TRAC-890.

** JIRA Ticket**: [TRAC-890](https://jira.lib.utk.edu/browse/TRAC-890)

# What does this Pull Request do?

Adds full display, brief display, and facets according to Rachel.

# What's new?
Everything was defined in the relevant SQL tables and exported so they can be remiported.

# How should this be tested?

1. Checkout the branch.
2. Vagrant destroy --force
3. Vagrant up
4. Add a record and update via Gsearch
5. Check Rresults

# Interested parties
@CanOfBees
